tests/Oryx.Integration.Tests/LocalDockerTests/MySqlIntegrationTests.cs

```diff
@@ -18,13 +18,13 @@ public MySqlIntegrationTests(ITestOutputHelper output, Fixtures.MySqlDbContainer
         {
         }
 
-        [Fact(Skip = "Bug 847845")]
+        [Fact]
```

Comment:
I will not check this PR in anytime soon. I am running these tests continuously on linux vm and if they are stable, will do

Reply:
I have run all the database tests around 30 times on my VM and they are running well without an issue, so I am merging this in.